packages/block-library/README.md

```diff
@@ -16,6 +16,14 @@ _This package assumes that your code will run in an **ES2015+** environment. If
 
 <!-- START TOKEN(Autogenerated API docs) -->
 
+<a name="registerBlock" href="#registerBlock">#</a> **registerBlock**
```

Comment:
I don't think that's a good API to expose, it's too confusing as it's the same as wp.blocks.registerBlock with minor differences (loading metadata), we should just duplicate it in edit-widgets instead.